crud/pom.xml

```diff
@@ -90,6 +90,11 @@
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-simple</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.redhat.lightblue.hook</groupId>
+            <artifactId>lightblue-audit-hook</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
@bserdar you're right. I saw lightblue-core in the subject and it threw me. Disregard.